end-to-end-tests/script/WithdrawFromInventory.s.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity >=0.8.20;
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+import { ResourceId, WorldResourceIdLib } from "@latticexyz/world/src/WorldResourceId.sol";
+import { IBaseWorld } from "@eveworld/world/src/codegen/world/IWorld.sol";
+import { InventoryItem } from "@eveworld/world/src/modules/inventory/types.sol";
+import { InventoryLib } from "@eveworld/world/src/modules/inventory/InventoryLib.sol";
+import { SmartStorageUnitLib } from "@eveworld/world/src/modules/smart-storage-unit/SmartStorageUnitLib.sol";
+import { FRONTIER_WORLD_DEPLOYMENT_NAMESPACE } from "@eveworld/common-constants/src/constants.sol";
+import { InventoryItemTableData, InventoryItemTable } from "@eveworld/world/src/codegen/tables/InventoryItemTable.sol";
+import { Utils as InventoryUtils } from "@eveworld/world/src/modules/inventory/Utils.sol";
+
+contract WithdrawFromInventory is Script {
+  using InventoryLib for InventoryLib.World;
+  using InventoryUtils for bytes14;
+
+  function run(address worldAddress) public {
+    StoreSwitch.setStoreAddress(worldAddress);
+    // Load the private key from the `PRIVATE_KEY` environment variable (in .env)
+    uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+    address invOwner = vm.addr(deployerPrivateKey);
+
+    // Start broadcasting transactions from the deployer account
+    vm.startBroadcast(deployerPrivateKey);
+    InventoryLib.World memory inventory = InventoryLib.World({
+      iface: IBaseWorld(worldAddress),
+      namespace: FRONTIER_WORLD_DEPLOYMENT_NAMESPACE
+    });
+
+    // uint256 smartObjectId = uint256(39041226111257929967735499841581212310485402913336957115444953092733099610999);
```

Comment:
Should this be commented out?

Reply:
it was test id from game. will remove it